README.md

```diff
@@ -1 +1,46 @@
-# TDS_ConfigService
\ No newline at end of file
+# TDS_ConfigService
+## Overview
+The `TDS_ConfigService` (aka Config Support Service) consists of two modules:
+
+* **client:** Contains the POJOs/classes needed for a consumer to interact with the Config Support Service
+* **service:** REST endpoints that provide TDS configuration data
+
+## Build
+* To build the **client**:
+  * `mvn clean install -f /path/to/client/pom.xml`
+* To build the **service**:
+  * `mvn clean install -f /path/to/service/pom.xml`
+* To build the service and run integration tests:
+  * `mvn clean install -Dintegration-tests.skip=false -f /path/to/service/pom.xml`
+
+## Integration Test Notes
+* Integeration tests are not run during a typical Maven build
+* To run integration tests during a build, refer to the **Build** section above
+
+### Database Migrations
+* The database migrations stored in `src/test/resources/db/migration` are only intended to support integration tests.  As such, they represent a small subset of the overall database schema for the `configs` and `session` databases.
+* Database Migrations are only executed when the `failsafe` plugin is configured to run.  By default, `failsafe` is configured to skip running integration tests when Maven builds the project.
+* [Flyway](https://flywaydb.org/) is used to execute the database migrations against an H2 in-memory database before the integration tests are executed. 
+
+### Naming Database Migration Files
+* Naming database migration files uses the following convention (note there are two underscores between the timestamp and the database name):
+
+V[***timestamp***]_\_[***database name***]\_[***DDL operation***]\_[***object name***].sql
+
+* To generate the timestamp, use use [http://www.unixtimestamp.com/](http://www.unixtimestamp.com/) and copy the UTC timestamp
+* Example file name: `V1472923547__configs_create_table_client_systemflags.sql`
```

Comment:
Suggest moving your example to be right under the version format on line 28.  It makes it easier to compare.